scripts/compare/flatten_mouse_files.pl

```diff
@@ -15,9 +15,11 @@
 # limitations under the License.
 
 
-=head1
+=pod
+
```

Comment:
Based on the convention we are following in other files, I would take the opportunity to fix this, replacing `=pod` by:
```
=cut

=head1 DESCRIPTION

```

Reply:
`DESCRIPTION` is a good idea